config/src/main/java/com/typesafe/config/impl/Tokenizer.java

```diff
@@ -299,7 +299,7 @@ private Token pullComment(int firstChar) {
         // chars JSON allows to be part of a number
         static final String numberChars = "0123456789eE+-.";
         // chars that stop an unquoted string
-        static final String notInUnquotedText = "$\"{}[]:=,+#`^?!@*&\\";
+        static final String notInUnquotedText = "$\"{}()[]:=,+#`^?!@*&\\";
```

Comment:
_(hello)_ previously parsed as _tokenUnquoted("(footrue)")_

Docs say .. 

_A sequence of characters outside of a quoted string is a string value if:
it does not contain "forbidden characters": '$', '"', '{', '}', '[', ']', ':', '=', ',', '+', '#', '`', '^', '?', '!', '@', '*', '&', '\' (backslash), or whitespace._

.. so of course no mention of () but then the behaviour of () outside "" isn't explicit